package.json

```diff
@@ -14,27 +13,59 @@
     "lint": "eslint \"src/**/*.ts\"",
     "lint:fix": "eslint \"src/**/**.{ts,tsx}\" --fix",
     "perf": "parcel perf/index.html",
-    "build:tsc": "tsc --project ./tsconfig.json",
+    "build:tsc": "tsc --emitDeclarationOnly --project ./tsconfig.json",
     "build:umd": "parcel build --target umd",
-    "build": "npm run clean && npm run build:tsc && npm run build:umd",
+    "build:react": "parcel build --dist-dir './dist/react' .'/src/react/index.ts'",
+    "build": "npm run clean && npm run build:tsc && parcel build",
```

Comment:
this doesn't call `build:react`, should it? otherwise `build:react` will never get invoked when deploying.